### PR TITLE
Update flash progress style

### PR DESCRIPTION
**Pull request by thundron**

### CODE REVIEW

File: lib/gui/app/components/drive-selector/target-selector.tsx [hunk 1]
```diff
@@ -132,13 +133,15 @@ export function TargetSelector(props: TargetSelectorProps) {
...
 	return (
-		<StepButton
-			primary
-			tabIndex={targets.length > 0 ? -1 : 2}
-			disabled={props.disabled}
-			onClick={props.openDriveSelector}
-		>
-			Select target
-		</StepButton>
+		<StepSelection>
```
Comment (zvin) [inline lib/gui/app/components/drive-selector/target-selector.tsx:136]: There is no need for this `<StepSelection>` tag.

File: lib/gui/app/components/progress-button/progress-button.tsx [hunk 2]
```diff
@@ -14,69 +14,133 @@
...
+		const { active, label, onCancel, type, percentage, warning } = this.props;
+		if (active) {
+			if (label.status !== 'starting') {
+				return (
+					<Txt align="left" color="white">
+						<Flex justifyContent="space-between">
+							<ProgressStatusLabel
+								label={label}
+								color={colors[type]}
+								style={{ fontSize: '16px', fontFamily: 'SourceSansPro' }}
```
Comment (zvin) [inline lib/gui/app/components/progress-button/progress-button.tsx:123]: This style should be defined in `ProgressStatusLabel`

File: lib/shared/units.ts [hunk 3]
```diff
@@ -24,11 +24,11 @@ export function bytesToMegabytes(bytes: number): number {
 	return bytes / MEGABYTE_TO_BYTE_RATIO;
 }
 
-export function bytesToClosestUnit(bytes: number): string | null {
+export function bytesToClosestUnit(bytes: number): string | undefined {
 	if (_.isNumber(bytes)) {
 		return prettyBytes(bytes);
 	}
-	return null;
+	return;
```
Comment (zvin) [inline lib/shared/units.ts:31]: Unneeded `return`

File: tests/gui/modules/progress-status.spec.ts [hunk 4]
```diff
@@ -17,7 +17,20 @@
...
+
+function getProgressLabel({
+	status,
+	...rest
+}: ProgressStatusType): ProgressStatusType {
+	return {
+		status,
+		...rest,
+	} as ProgressStatusType;
+}
```
Comment (zvin) [inline tests/gui/modules/progress-status.spec.ts:33]: This function does nothing.

Comment (thundron) [reply tests/gui/modules/progress-status.spec.ts:33]: leftover of some experiments, chai was complaining about { status: '...' } not being equal to { status: '...' } and I started going nuts

File: lib/gui/app/pages/main/Flash.tsx [hunk 5]
```diff
@@ -219,58 +231,54 @@ export class FlashStep extends React.Component<FlashStepProps, FlashStepState> {
...
-		const isFlashing = flashState.isFlashing();
-		const flashErrorCode = flashState.getLastFlashErrorCode();
-		return (
-			<>
-				<div className="box text-center">
-					<div className="center-block">
-						<SVGIcon
-							paths={['flash.svg']}
-							disabled={this.props.shouldFlashStepBeDisabled}
+		setErrorMessage(await flashImageToDrive(goToSuccess, source));
```
Comment (zvin) [inline lib/gui/app/pages/main/Flash.tsx:234]: This is broken (probably because of a rebase)

Comment (thundron) [reply lib/gui/app/pages/main/Flash.tsx:234]: Yep I'm fixing it, mainly for the switch from arrow function to class

**Pull request closed without merging**